core/testing/go.mod

```diff
@@ -6,12 +6,5 @@ replace cosmossdk.io/core => ../
 
 require (
 	cosmossdk.io/core v0.12.0
-	github.com/cosmos/gogoproto v1.6.0
```

Comment:
The dependency to gogo may be not too bad here, as it makes sense to test the dummy services

Reply:
since its using structural typing its the same without it too